### DISCUSSION TITLE
Simplify Custom Rules

### DISCUSSION THREAD

**Discussion by daz-codes** [josdejong/mathjs#2481] [Q&A] [answered]
I'd like to add a couple of custom rules to the simplify function, one to stop it factorising expressions and another to stop negative terms being placed inside parentheses:

```
const equation = "-4x^1 - 4"
console.log(simplify(equation,rules).toString())
<<  "-(4 * (x + 1))"

const rules = ['(n1+n2)*n3 -> n1*n3 + n2*n3','-(n1) -> -n1']
console.log(simplify(equation,rules).toString())
<< "-4 x ^ 1 - 4"
```

Unfortunately these new rules seem to remove all the other default rules, so the power of -1 is now showing when previously it didn't. Is there any way of adding new rules while keeping the default rules?

I'd like the output to be: `"-4x-4"`

Thanks!

**Comment by josdejong** [comment, marked as answer]
The build in rules are exposes as `math.simplify.rules`, you can copy that array and add new entries to it (or mix and match the way you).

**Reply by daz-codes** [reply]
Thanks, that's really useful to know.
